src/components/PostsWrap.tsx

```diff
@@ -1,44 +1,47 @@
-import React, { useState, useEffect } from 'react';
-import LayoutToggle from './LayoutToggle';
-import Modal from './Modal';
-import SecondaryNav from './SecondaryNav';
-import GridDisplay from './GridDisplay';
-import ListDisplay from './ListDisplay';
-import { fetchRecommendations } from '../lib/supabase';
-import useSupabaseAuth from '../hooks/useSupabaseAuth';
-import locationsHash from '../lib/locationsHash';
-import { useLocation, useSearchParams } from 'react-router-dom';
+import React, { useState, useEffect } from "react";
+import LayoutToggle from "./LayoutToggle";
+import Modal from "./Modal";
+import SecondaryNav from "./SecondaryNav";
+import GridDisplay from "./GridDisplay";
+import ListDisplay from "./ListDisplay";
+import { fetchRecommendations } from "../lib/supabase";
+import useSupabaseAuth from "../hooks/useSupabaseAuth";
+import locationsHash from "../lib/locationsHash";
+import { useLocation, useSearchParams } from "react-router-dom";
+import HotRepositories from "./HotRepositories";
 
-interface PostWrapProps{
-  textToSearch: string
+interface PostWrapProps {
+  textToSearch: string;
 }
 
-const parseLimitValue = (limit: string | null) : number => {
-  if(!limit) {
+const parseLimitValue = (limit: string | null): number => {
+  if (!limit) {
     return 25;
   }
-  const value = parseInt(limit)
-  if(isNaN(value) || value <= 0) {
+  const value = parseInt(limit);
+  if (isNaN(value) || value <= 0) {
     return 25;
   }
-  if(value > 100) {
+  if (value > 100) {
     return 125;
   }
   return value;
-}
+};
 
 const PostsWrap = ({ textToSearch }: PostWrapProps): JSX.Element => {
   const [isGrid, setIsGrid] = useState(true);
+  // const [activeLink, setActiveLink] = useState("popular");
+  // const [limit, setLimit] = useState(25);
```

Comment:
What is happening here?

Reply:
I commented out this code when I was resolving some merge conflicts I was having when I want to make a PR earlier. I will remove the commented code complete and also check to make sure I don't have any other unnecessary comments lying around in the code base